services/ui/public/interfaces/window_tree.mojom

```diff
@@ -370,6 +371,12 @@ interface WindowTree {
 // calling SetWindowBounds(), connection 1 does not receive
 // OnWindowBoundsChanged().
 interface WindowTreeClient {
+  // Called to notify about the real AcceleratedWidget associated with a
+  // window. This is called only when mus is not hosting viz, and requires the
+  // window manager to instead host viz.
```

Comment:
nit: window manager to(?) instead host viz

a bit hard to read ^^

Reply:
done